### ISSUE TITLE
Unable to calculate max please try again

### ISSUE THREAD

**Issue by timstackblock** [blockstack/blockstack-app#2103]
When attempting to send STX I encountered this issue in prod.

I was trying to send the Max amount in my wallet but this error message appeared.

<img width="315" alt="Screen Shot 2021-12-25 at 4 25 20 PM" src="https://user-images.githubusercontent.com/49165468/147393835-6b28f32d-431a-4b3c-acb3-ebaacb61fe85.png">

**Comment by fbwoolf** [comment]
@timstackblock that is expected behavior when there is no fee estimation to use to perform the calculation, was a fee set at that point?

The conditional for showing that toast is `isLoadingFee && isStx`

**Comment by timstackblock** [comment]
Yes everything loaded, there were multiple fee options, I selected the standard option everything looked as if it had loaded but when I clicked the send button I got this error message. I repeated the same use case 3 times it failed 2 times and it worked on the third try.

**Comment by timstackblock** [comment]
I'll get a video for you shortly

**Comment by fbwoolf** [comment]
Ok, great, thx ...not sure how that could happen so I'll try to repro.

**Comment by markmhendrickson** [comment]
Have either of you seen this recently?

**Comment by Eshwari007** [comment]
Not lately from my end
